tests/phpunit/tests/admin/includesScreen.php

```diff
@@ -440,6 +435,7 @@ function test_options() {
 			'option'  => $option,
 		);
 
+		set_current_screen( 'front' );
```

Comment:
It might be clearer and more future-proof for these three instances (lines 307, 369, 438) to use something other than `front` for the screen name. Using `front` sets the `WP_Screen::$in_admin` property to `false` and you could argue that screen options and help text shouldn't operate in that case.

Reply:
@johnbillion I like that idea. And you're right that technically `front` would not have screen options or help text. Switching these 3 to a core screen => `edit.php`. 

Done in commit https://github.com/WordPress/wordpress-develop/pull/1380/commits/35808948e8b7db54b77506a2eeb021a2ee97a4b8.